### PR TITLE
6308 mdbc integration

### PR DESCRIPTION
**Pull request by h0st**
Single mariadb enterprise and community recipe and one for maxscale

### CODE REVIEW

File: recipes/mdbrepos.rb [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
-  template "/etc/zypp/repos.d/mariadb.repo.template" do
+  template "/etc/zypp/repos.d/mariadb.repo" do
     source "mariadb.suse.erb"
     action :create
   end
-  release_name = "test -f /etc/os-release && cat /etc/os-release | grep '^ID=' | sed s/'^ID='//g | sed s/'\"'//g || if cat /etc/SuSE-release | grep Enterprise &>/dev/null; then echo sles; else echo opensuse; fi"
+  release_name = "if cat /etc/SuSE-release | grep Enterprise &>/dev/null; then echo sles; else echo opensuse; fi"
   execute "Change suse on sles repository" do
-    command "cat /etc/zypp/repos.d/mariadb.repo.template | sed s/PLATFORM/$(" + release_name + ")/g > /etc/zypp/repos.d/mariadb.repo"
+  	command "cat /etc/zypp/repos.d/mariadb.repo | sed s/PLATFORM/$(" + release_name + ")/g > /etc/zypp/repos.d/mariadb.repo"
```
Comment (ferustigris) [inline recipes/mdbrepos.rb]: remove tab

File: recipes/install_maxscale.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+include_recipe "mariadb::maxscale_repos"
```
Comment (ferustigris) [inline recipes/install_maxscale.rb]: there are different products - MDBE and max scale, so it is better to store it in separated coockbook

File: attributes/default.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#
+# Ubuntu key for Enterprise/Community repository
+default["ubuntu"]["key"] = "0xcbcb082a1bb943db"					# Community 
+#default["ubuntu"]["key"] = "0xce1a3dd5e3c94f49"				# Enterprise
+
+default["maria"]["version"] = "10.0"
+
+# mariadb repo ubuntu/debian/mint
+default["maria"]["repo"] = "http://mirror.mephi.ru/mariadb/repo"		# Community
+#default["maria"]["repo"] = "https://code.mariadb.com/mariadb-enterprise"	# Enterprise
```
Comment (ferustigris) [inline attributes/default.rb]: It looks not convenience to uncomment 20+ lines if you are going to install MDBE. M.b. initialize default variables depending on param?
'mariadb::distribution': ['enterprise' | 'community']?

What do you think about of it?

File: recipes/install_maxscale.rb (hunk 2 above)
Comment (h0st) [inline recipes/install_maxscale.rb]: Ok, i have separate cookbook for maxscale, it in mdbci template/examples directory.

File: recipes/mdbrepos.rb (hunk 1 above)
Comment (h0st) [inline recipes/mdbrepos.rb]: ok

File: attributes/default.rb (hunk 3 above)
Comment (h0st) [inline attributes/default.rb]: Yes, i am agree. How can i realize it?
This params will be init for each node in it role. Node with community, node with enterprise and so on.

**Comment by h0st** [comment]
I made a mistake with repo and make pull request to your's repo. So i think, we will fix it and integrate.

File: attributes/default.rb (hunk 3 above)
Comment (ferustigris) [inline attributes/default.rb]: Just idea:
node.set_unless["maria"]["distribution"] = "enterprise"
if default["maria"]["distribution"] == "enterprise"
  include_recipe "mariadb::init_enterprise"
else
  include_recipe "mariadb::init_community"
end

init_community.rb:
default["maria"]["family"] = "/"

init_enterprise.rb:
default["maria"]["family"] = "/yum/"

**Pull request closed without merging**